pkg/api/v1/types_swagger_doc.go

```diff
@@ -0,0 +1,1164 @@
+		"involvedObject": "The object that this event is about.",
+		"reason":         "This should be a short, machine understandable string that gives the reason for the transition into the object's current status.",
+		"message":        "A human-readable description of the status of this operation.",
+		"source":         "The component reporting this event. Should be a short machine understandable string.",
+		"firstTimestamp": "The time at which the event was first recorded. (Time of server receipt is in TypeMeta.)",
+		"lastTimestamp":  "The time at which the most recent occurrence of this event was recorded.",
+		"count":          "The number of times this event has occurred.",
+	}
+}
+
+func (EventList) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "EventList is a list of events.",
+		"metadata": "Standard list metadata. More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#types-kinds",
+		"items":    "List of events",
+	}
+}
+
+func (EventSource) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":          "EventSource contains information for an event.",
+		"component": "Component from which the event is generated.",
+		"host":      "Host name on which the event is generated.",
+	}
+}
+
+func (ExecAction) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":        "ExecAction describes a \"run in container\" action.",
+		"command": "Command is the command line to execute inside the container, the working directory for the command  is root ('/') in the container's filesystem. The command is simply exec'd, it is not run inside a shell, so traditional shell instructions ('|', etc) won't work. To use a shell, you need to explicitly call out to that shell. Exit status of 0 is treated as live/healthy and non-zero is unhealthy.",
+	}
+}
+
+func (GCEPersistentDiskVolumeSource) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":          "GCEPersistentDiskVolumeSource represents a Persistent Disk resource in Google Compute Engine.\n\nA GCE PD must exist and be formatted before mounting to a container. The disk must also be in the same GCE project and zone as the kubelet. A GCE PD can only be mounted as read/write once.",
+		"pdName":    "Unique name of the PD resource in GCE. Used to identify the disk in GCE. More info: http://releases.k8s.io/HEAD/docs/user-guide/volumes.md#gcepersistentdisk",
+		"fsType":    "Filesystem type of the volume that you want to mount. Tip: Ensure that the filesystem type is supported by the host operating system. Examples: \"ext4\", \"xfs\", \"ntfs\". More info: http://releases.k8s.io/HEAD/docs/user-guide/volumes.md#gcepersistentdisk",
+		"partition": "Partition on the disk to mount (e.g., '1' for /dev/sda1). If omitted, kubelet will attempt to mount the device name. Ex. For /dev/sda1, this field is \"1\", for /dev/sda, this field is 0 or empty. More info: http://releases.k8s.io/HEAD/docs/user-guide/volumes.md#gcepersistentdisk",
```

Comment:
Revision: 
"The partition in the volume that you want to mount. If omitted, the default is to mount by volume name. Examples: For volume /dev/sda1, you specify the partition as \"1\". Similarly, the volume partition for /dev/sda is \"0\" (or you can leave the property empty)....."